draft-ietf-taps-interface.md

```diff
@@ -1288,16 +1303,16 @@ Connection properties include:
 
 * The status of the Connection, which can be one of the following:
   Establishing, Established, Closing, or Closed.
```

Comment:
See my comments on the `Closing` issue—I do think that this is of no benefit to the application, and doesn't provide a meaningful signal in the state machine. We can certainly prohibit send and receive after close, but that doesn't necessarily align with a closing state when things are asynchronous.

Reply:
ISTM there's probably another PR for the closing state issue once discussion completed; I won't address this here.